SignalShareExtension/ShareViewController.swift

```diff
@@ -310,7 +334,7 @@ public class ShareViewController: UINavigationController, SAELoadViewDelegate {
         Logger.flush()
     }
 
-    // MARK: SAELoadViewDelegate
+    // MARK: SAELoadViewDelegate, SAEFailedViewDelegate
```

Comment:
I propose that we create a generic `SAEViewDelegate` that  can be used by all subviews to cancel or complete.  It might be nice to add a base class `SAEViewController` for all SAE view controllers that DRYs up common logic.